### PR TITLE
feat: add featured collections component

### PR DESCRIPTION
**Pull request by shahin-hq**
## Summary
Closes: [[articles] add featured collections componen](https://app.clickup.com/t/8678v9pvd)

## Checklist

-   [x] I checked my UI changes against the design and there are no notable differences
-   [x] I checked my UI changes for any responsiveness issues
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] I added a storybook entry for the component that was added _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [x] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: resources/js/Pages/Articles/Components/ArticleCard.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                    <CollectionCircles />
+                </div>
+            </div>
+        </div>
+    </div>
+);
+
+const CollectionCircles = () => (
+    <div className="flex shrink-0 grow flex-nowrap">
+        {collections.slice(0, 15).map((collection, index) => (
```
Comment (alfonsobries) [inline resources/js/Pages/Articles/Components/ArticleCard.tsx:85]: max amount is 8 I think

**Comment by ItsANameToo** [comment]
now that https://github.com/ArdentHQ/dashbrd/pull/144 is merged, this can be combined with those implemented article cards

**Review by ItsANameToo — CHANGES_REQUESTED** [review]
can you add some variations to storybook so it showcases the available scenarios, e.g.:
- 1 collection (single article card)
- 5 collections (single article card)
- 8 collections (single article card)
- grid with 8 cards where they have various amount of collections to see how it looks in a more real-life scenario

**Comment by shahin-hq** [comment]
@ItsANameToo I made changes as you suggested:

`ArticleListItem`:

image

`ArticleCard`

image

`FeaturedCollections` - as a unit

image

**Pull request merged**